backend/functions/lib/adapters/gateways/cases.cosmosdb.repository.ts

```diff
@@ -96,9 +96,11 @@ export class CasesCosmosDbRepository implements CasesRepository {
           .database(this.cosmosConfig.databaseName)
           .container(this.containerName)
           .items.create(itemToCreate);
+        context.logger.info(MODULE_NAME, 'Created the following resource:', resource);
         return resource;
       } catch (e) {
         if (!isPreExistingDocumentError(e)) {
+          context.logger.info(MODULE_NAME, 'Item already exists: ', itemToCreate);
```

Comment:
Concern about logging content of object here. We could stick with an id and/or partition key

Reply:
Agreed

Reply:
Yep. Same as above. Leaking the entire object.

Reply:
We added functionality to remove any properties that are disallowed. The objects being logged here are generics, so we don't know what properties they have and which if any are inappropriate for logging.